### PR TITLE
Blog post for release of v1.0.0

### PR DESCRIPTION
**Pull request by jennybc**
http://rpubs.com/jennybc/readxl-v1-0-0

### CODE REVIEW

**Comment by jennybc** [comment]
I followed @jimhester's [xml2 post](https://blog.rstudio.org/2017/01/24/xml-1-1-1/) as a template.

File: vignettes/releases/readxl-1.0.0.Rmd [hunk 1]
```diff
@@ -4,6 +4,67 @@ title: "readxl v1.0.0"
...
+You can install it with:
+
+```{r eval = FALSE}
+install.packages("readxl")
+```
+
+As well as fixing many bugs, this release:
+
+  * Allows you to target specific cells for reading, in a variety of ways
+  * Adds two new column types, `"list"` for data of disparate type, and `"logical"`
```
Comment (hadley) [inline vignettes/releases/readxl-1.0.0.Rmd]: Flip the order here? (logical then list)

File: vignettes/releases/readxl-1.0.0.Rmd [hunk 2]
```diff
@@ -4,6 +4,67 @@ title: "readxl v1.0.0"
...
+You can see a full list of changes in the [release notes](http://readxl.tidyverse.org/news/index.html). This is the first release maintained by Jenny Bryan.
+  
+## Specifying the data rectangle
+
+In an ideal world, data would live in a neat rectangle in the upper left corner of a spreadsheet. But spreadsheets often serve multiple purposes for users with different priorities. It is common to encounter several rows of notes above or below the data, for example. The new `range` argument provides a flexible interface for describing the data rectangle, including Excel-style ranges and row- or column-only ranges.
+
+```{r}
+library(readxl)
+read_excel(readxl_example("deaths.xlsx"), range = "arts!A5:F15")
+read_excel(readxl_example("deaths.xlsx"),
```
Comment (hadley) [inline vignettes/releases/readxl-1.0.0.Rmd]: Tidy style indenting please?

File: vignettes/releases/readxl-1.0.0.Rmd [hunk 3]
```diff
@@ -4,6 +4,67 @@ title: "readxl v1.0.0"
...
+           sheet = "other", range = cell_rows(5:15))
+```
+
+There is also a new argument `n_max` that limits the number of data rows read from the sheet. It is an example of [readxl][readxl]'s evolution towards a [readr](http://readr.tidyverse.org)-like interface.
```
Comment (hadley) [inline vignettes/releases/readxl-1.0.0.Rmd]: Link to geometry article?

**Pull request merged**